finder-app/finder.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+# recommended by deekshith patil on office hours
+
+if [ $# -ne 2 ];
+then
+   echo "Arguments requested not found, exiting...."
```

Comment:
@srki3050 think about how you could make this message more descriptive to the user to help explain how to resolve.